.github/workflows/python-release.yaml

```diff
@@ -26,11 +24,6 @@ jobs:
       - name: Install dependencies
         run: yum install -y openssl-devel
 
-      - name: ✍️ auth
-        uses: google-github-actions/auth@v0
-        with:
-          credentials_json: ${{ secrets.TOKENIZER_SERVICE_ACCOUNT_KEY }}
-
```

Comment:
Can we also remove the `Build and audit wheels` step below, and the `build-wheels.sh` script? Seems that `build-wheels.sh` compiles and uploads the tokenizer binary, but unsure if this is needed.

Reply:
Ah i removed build-wheels.sh  but didn't catch the CI step that used it. Removed it!